plugins/ImageGraph/API.php

```diff
@@ -238,8 +238,16 @@ public function get(
 
             $ordinateColumns = array();
             if (empty($columns)) {
-                $ordinateColumns[] =
-                    empty($reportColumns[self::DEFAULT_ORDINATE_METRIC]) ? key($metadata['metrics']) : self::DEFAULT_ORDINATE_METRIC;
+                if (!empty($metadata['metrics']) && empty($reportColumns[self::DEFAULT_ORDINATE_METRIC])) {
+                    $ordinateColumns[] = key($metadata['metrics']);
+                } else {
```

Comment:
Now it's not doing the same as before anymore. `self::DEFAULT_ORDINATE_METRIC` should still be used if it is available:
```suggestion
                if (!empty($reportColumns[self::DEFAULT_ORDINATE_METRIC])) {
                    $ordinateColumns[] = self::DEFAULT_ORDINATE_METRIC];
                } else if (!empty($metadata['metrics'])) {
                    $ordinateColumns[] = key($metadata['metrics']);
                } else {
```